File: src/main/java/com/linebot/entity/TrainDelayLog.java
package com.linebot.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@Builder
@Entity
@Table(name = "train_delay_log")
@NoArgsConstructor
@AllArgsConstructor
public class TrainDelayLog {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;
    @Column(name = "train_name")
    private String trainName;
    @Column(name = "year_month")
    private String yearMonth;
    @Column(name = "day")
    private int day;
    @Column(name = "day_of_week")
    private int dayOfWeek;
    @Column(name = "hour")
    private int hour;
    @Column(name = "minute")
    private int minute;
    @Column(name = "exec_time")
    private Date execTime;

}

File: src/main/kotlin/com/linebot/entity/TrainDelayLog.kt
package com.linebot.entity

import lombok.Data
import java.util.*
import javax.persistence.*

@Data
@Entity
@Table(name = "train_delay_log")
data class TrainDelayLog(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        @Column(name = "id")
        var id: Int? = null,

        @Column(name = "train_name")
        var trainName: String? = null,

        @Column(name = "year_month")
        var yearMonth: String? = null,

        @Column(name = "day")
        var day: Int = 0,

        @Column(name = "day_of_week")
        var dayOfWeek: Int = 0,

        @Column(name = "hour")
        var hour: Int = 0,

        @Column(name = "minute")
        var minute: Int = 0,

        @Column(name = "exec_time")
        var execTime: Date? = null
)